### ISSUE TITLE
I can not Upload firmware using "System Refresh"

### ISSUE THREAD

**Issue by williamesp2015** [AdySan/ESPSocket#2]
Great job!!!. However I could't upload after connecting to the Websocket server in the "System Refresh". It shows the firmware.bin in the screen but progress bar has no progress.
Thank you

![image](https://cloud.githubusercontent.com/assets/12815126/17736306/41dc2dbc-649c-11e6-8d07-d3b125757aa2.png)

**Comment by AdySan** [comment]
I don't remember if the progress bar did actually work, does it actually write the .bin file to flash? 
Can you enable the console in Developer tools and check if there are any errors?

![image](https://cloud.githubusercontent.com/assets/744810/17756538/5b6362de-6495-11e6-91fc-30d202453aa2.png)

**Comment by williamesp2015** [comment]
seemd it try to upload to url not refresh firmware by OTA
http://adyesp8266.local/edit net::ERR_NAME_NOT_RESOLVED

**Comment by AdySan** [comment]
yeah, maybe something has changed in OTA since I made this, OR mdns is not
working, can you try replacing adyesp8266.local with the IP address?
